#// Структура сосдержит всю информацию по мотору на основании данных энкодера
	float32 rpsSet            # Текущая скорость вращения ( обороты в секунду)
	float32 rpsEncod            # Текущая скорость вращения ( обороты в секунду)